tests/docker-images/statestore-image/bin/bkctl

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
Are these different from the scripts that are already in top level `./bin` directory?

Reply:
Yeah. It's stripped down version of that.
IN the bin it builds the classpath like the following. 
`CLI_MODULE_NAME="(org.apache.bookkeeper-)?bookkeeper-tools"
CLI_MODULE_HOME=${BK_HOME}/${CLI_MODULE_PATH}

# find the module jar
CLI_JAR=$(find_module_jar ${CLI_MODULE_PATH} ${CLI_MODULE_NAME})

# set up the classpath
CLI_CLASSPATH=$(set_module_classpath ${CLI_MODULE_PATH})`  While in the docker image we know where the jars are put so we do not have to search for module jar etc. 

Therefore we can directly use the classpath.
`CLI_CLASSPATH=${BK_HOME}/lib/*`

Reply:
Same things for conf files.

Reply:
Conf files I think are ok, but the bin scripts it would be nice to have tests to use the same one (so that we can actually test them). Do you see a way to support the 2 different cases?

Reply:
I understand that. It can be done If we name the jars accordingly.